ui/app/scm-repositories/components/pnc-internal-scm-repository-link/pnc-internal-scm-repository-link.html

```diff
@@ -0,0 +1,23 @@
+<!--
+
+    JBoss, Home of Professional Open Source.
+    Copyright 2014-2020 Red Hat, Inc., and individual contributors
+    as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<span>{{ $ctrl.internalScmRepositoryUrl || 'Empty' }}</span>
+<span ng-if="$ctrl.internalScmRepositoryUrl">
+  (<a ng-href="{{ $ctrl.parseInternalRepoLink() }}" target="_blank" id="internal-url" class="form-control-static">gerrit</a>)
```

Comment:
I would suggest that you may change the symbol from `(gerrit)` to `[ Gerrit ]` so that it would be compatible with all other links on the build details page.

Reply:
![image](https://user-images.githubusercontent.com/6870455/109507743-f3dd9400-7a96-11eb-86cc-ca46703b8ac3.png)

Reply:
Yeah, that's good idea. Definitely looks better this way. Thanks :)